docs/install/tutorials/rpi.rst

```diff
@@ -269,9 +269,14 @@ Installing Kolibri
       sudo su -c 'echo "deb http://ppa.launchpad.net/learningequality/kolibri/ubuntu xenial main" > /etc/apt/sources.list.d/learningequality-ubuntu-kolibri-xenial.list'
       sudo apt-key adv --keyserver hkp://keyserver.ubuntu.com:80 --recv-keys DC5BAA93F9E4AE4F0411F97C74F88ADB3194DD81
       sudo apt update
-      sudo apt install kolibri
 
-   When asked questions during the installation, it is recommended that you use the default ``pi`` user for running Kolibri because it will have access to USB devices.
+#. Install ``kolibri`` and ``kolibri-server``:
+
+   .. code-block:: bash
+
+      sudo apt install kolibri kolibri-server
+
+   When prompted, use the default TCP port ``8080`` for Kolibri because port ``80`` will be used by Nginx :ref:`later in this tutorial <nginx_custom_domain>`. Use the default ``pi`` user so that it will have access to USB devices.
```

Comment:
That's not true with kolibri-server, right? It spins up its own nginx conf using the requested port, doesn't it? I wouldn't imagine ewe'd want to then also add an additional nginx layer?

Reply:
We are adding an additional Nginx layer in both cases. My reasoning was to keep `8080` for the `kolibri-server` nginx configuration and add another one in the tutorial to make the recipe more evident, and that one is for port `80`. It doesn't cause any extra load to the system to have an additional nginx configuration since it's just a proxy to the same uwsgi socket.